nodebuilder/pruner/module.go

```diff
@@ -34,31 +33,37 @@ func ConstructModule(tp node.Type, cfg *Config) fx.Option {
 		// This is necessary to invoke the pruner service as independent thanks to a
 		// quirk in FX.
 		fx.Invoke(func(_ *pruner.Service) {}),
-		fx.Provide(func(store *eds.Store) pruner.Pruner {
-			return full.NewPruner(store)
-		}),
 	)
 
 	switch tp {
 	// TODO: Eventually, light nodes will be capable of pruning samples
 	//  in which case, this can be enabled.
```

Comment:
You can remove this TODO. 

I think a rebase artifact could have led to some of the changes missing, like this TODO, because I remember removing it. This may also explain why you could not run the Service(as per PR description), but I definitely could.